amd_openvx_extensions/amd_migraphx/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2017 - 2022 Advanced Micro Devices, Inc. All rights reserved.
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#  
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#  
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+cmake_minimum_required(VERSION 3.5)
+project(vx_amd_migraphx)
+
+set(CMAKE_CXX_STANDARD 14)
+
+list (APPEND CMAKE_PREFIX_PATH ${ROCM_PATH}/hip ${ROCM_PATH})
+
+if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
+    if(NOT DEFINED ENV{HSA_PATH})
+        set(HSA_PATH ${ROCM_PATH}/hsa)
+    else()
+        set(HSA_PATH $ENV{HSA_PATH})
+    endif()
+    if(NOT DEFINED HIP_PATH)
+        if(NOT DEFINED ENV{HIP_PATH})
+            set(HIP_PATH ${ROCM_PATH}/hip)
+        else()
+            set(HIP_PATH $ENV{HIP_PATH})
+        endif()
+    endif()
+
+    find_package(HIP QUIET)
+
+    list(APPEND PACKAGE_DEPENDS PACKAGE HIP)
+
+    find_package(migraphx QUIET)
+
+    include_directories(../../amd_openvx/openvx/include
+    ${HIP_PATH}/include ${HSA_PATH}/include
+    include)
+
+    list(APPEND SOURCES
+            src/kernels.cpp
+            src/migraphx_node.cpp
+        )
+
+    if (HIP_FOUND AND migraphx_FOUND)
+        message("-- ${Green}${PROJECT_NAME} -- Building with HIP backend${ColourReset}")
+        set(ENABLE_HIP 1)
+        add_definitions(-DENABLE_HIP=${ENABLE_HIP} -D__HIP_PLATFORM_AMD__)
+        link_directories(${HIP_PATH}/lib)
```

Comment:
probably we need to change to target_xxx definitions as per new CMake compliance

Reply:
@rrawther as discussed, we'll change the CMake for all projects in a separate PR.